rfcs/text/0041-asset-integration.md

```diff
@@ -165,14 +166,139 @@ There are many sources of asset inventory repositories. In the mid-term, we are
 * MS Intune
 * ServiceNow Asset CMDB
 
+### Examples of source data:
+
+#### Subset of User fields from Okta:
+
+```json
+{
+  "@timestamp": "2023-07-04T09:57:19.786056-05:00",
+  "event": {
+    "action": "user-discovered"
+  },
+  "okta": {
+    "id": "userid",
+    "status": "RECOVERY",
+    "created": "2023-06-02T09:33:00.189752+09:30",
+    "activated": "0001-01-01T00:00:00Z",
+    "statusChanged": "2023-06-02T09:33:00.189752+09:30",
+    "lastLogin": "2023-06-02T09:33:00.189752+09:30",
+    "lastUpdated": "2023-06-02T09:33:00.189753+09:30",
+    "passwordChanged": "2023-06-02T09:33:00.189753+09:30",
+    "type": {
+      "id": "typeid"
+    },
+    "profile": {
+      "login": "name.surname@example.com",
+      "email": "name.surname@example.com",
+      "firstName": "name",
+      "lastName": "surname"
+    },
+    "credentials": {
+      "password": {},
+      "provider": {
+        "type": "OKTA",
+        "name": "OKTA"
+      }
+    },
+    "_links": {
+      "self": {
+        "href": "https://localhost/api/v1/users/userid"
+      }
+    }
+  },
+  "user": {
+    "id": "userid"
+  },
+  "labels": {
+    "identity_source": "okta-1"
+  }
+}
+```
+
 <!--
 Stage 2: Included a real world example source document. Ideally this example comes from the source(s) identified in stage 1. If not, it should replace them. The goal here is to validate the utility of these field changes in the context of a real world example. Format with the source name as a ### header and the example document in a GitHub code block with json formatting, or if on the larger side, add them to the corresponding RFC folder.
 -->
 
+
 <!--
 Stage 3: Add more real world example source documents so we have at least 2 total, but ideally 3. Format as described in stage 2.
 -->
 
+### Examples of Real-world mapping: 
+
+#### Mapping User object from Okta (partial):
```

Comment:
Instead of the pipeline, is it possible to include the final, ECS mapped event?

It's a really helpful aid for understanding how these proposed fields will be used to map a data source.

Reply:
@ebeahan I have updated the 'Examples of Real-world mapping' section with ECS mappings.